docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "2"
+services:
+  backup-service:
+    container_name: backup-service
+#    restart: always
+    build: .
+    ports:
+      - "8000:8000"
+    links:
+      - mongo
+  mongo:
+    container_name: mongo
+    image: mongo
+    volumes:
+      - ./data:/data/db
+    ports:
```

Comment:
There is no need to export this service to the host, it will be accessible by the `backup-service` anyway without polluting host network